sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -52,6 +93,28 @@ export interface AspectSentiment {
 
 export { AzureKeyCredential }
 
+// @public
+export type BeginAnalyzeHealthcareOperationState = PollOperationState<PaginatedHealthcareEntities>;
+
+// @public
+export interface BeginAnalyzeHealthcareOptions {
+    // (undocumented)
```

Comment:
Is there a way for the user to specify `top` and `skip`?

Reply:
the user can specify the size of the page in `byPage` method. You can find examples in the tests for pagination in `apiKey.spec.ts`.